frontend/apps/core-app/src/components/AuthWrapper/useTokenResponse.ts

```diff
@@ -27,7 +27,11 @@ const getStoredTokenResponse = (): TokenResponse | undefined => {
 const storeTokenResponse = (tokenResponse: TokenResponse) => {
   try {
     if (Platform.OS !== 'web') return;
-    sessionStorage.setItem('tokenResponse', JSON.stringify(tokenResponse));
+    if (tokenResponse) {
```

Comment:
might be useful to write a test for this

Reply:
@neb42 are you planning to write a test here, or do you consider this PR done?

Reply:
Having trouble mocking session storage. I think the logic here is simple enough to leave it